[BUG] SIGSEGV panic from update flow

Experiencing panic on prerelease binaries. 

**Expected Cause**: The repository is private and the flow assumes it to be public and available.  

**Output**:
```
$ ./privado version
Privado CLI: Version v2.0-prerelease-test (linux-amd64) 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x7dad8a]

goroutine 1 [running]:
github.com/Privado-Inc/privado-cli/cmd.checkForUpdate()
        /github/workspace/cmd/update.go:32 +0x6a
github.com/Privado-Inc/privado-cli/cmd.version(0xc2e940?, {0xc66478?, 0x0?, 0x0?})
        /github/workspace/cmd/version.go:29 +0x135
github.com/spf13/cobra.(*Command).execute(0xc2e940, {0xc66478, 0x0, 0x0})
        /go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:876 +0x67b
github.com/spf13/cobra.(*Command).ExecuteC(0xc2e1c0)
        /go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:990 +0x3b4
github.com/spf13/cobra.(*Command).Execute(...)
        /go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:918
github.com/Privado-Inc/privado-cli/cmd.Execute()
        /github/workspace/cmd/root.go:23 +0x35
main.main()
        /github/workspace/main.go:29 +0x1c
```

--- a/cmd/update.go
+++ b/cmd/update.go
@@ -27,9 +27,10 @@ func checkForUpdate() (hasUpdate bool, updateMessage string, err error) {
 		return false, "", nil
 	}
 
-	// get release info
+	// get release info (nil when not available)
 	releaseInfo, err := utils.GetLatestReleaseFromGitHub(config.AppConfig.PrivadoRepositoryName)
-	if err != nil || releaseInfo.TagName == "" || releaseInfo.PublishedAt == "" {
+
+	if err != nil || releaseInfo == nil || releaseInfo.TagName == "" || releaseInfo.PublishedAt == "" {
 		return false, "", err
 	}
 

